docs/pulplift.md

```diff
@@ -83,7 +84,7 @@ vagrant up pulp3-sandbox-centos7
 Source boxes, such as `pulp3-source-centos7` can be used to do an install of pulp with developer tools & helper scripts/aliases for developers.
 
 The example [configuration](#Configuration) requires that the `pulpcore` git repo directory and plugin git repo directories (e.g. `pulp_file`) exist in folders,
-under the same parent folder as `pulplift`, so they can be mounted on the box.
+under the same parent folder as `pulp_installer`, so they can be mounted on the box.
```

Comment:
```suggestion
under the same parent folder as `pulp_installer`, so they will be mounted on the box.
```

Reply:
I will make the suggested changes and get an update available shortly. Thanks for the review.

Reply:
Thank you very much! I'll merge now.